### PR TITLE
Merge the OCS intro pages and RBAC 

### PR DESCRIPTION
**Pull request by Kitatine**
with node_modules deleted

### CODE REVIEW

File: tides-server/pkg/handler/user_api.go [hunk 1]
```diff
@@ -10,7 +12,45 @@ import (
...
+const OFFICIAL_EMAIL = "cloudtest2021abc@gmail.com"
+const OFFICIAL_PASSWORD = "cloudtides"
+
+// ResetPasswordHandler is the API for /users/reset POST
+func ResetPasswordHandler(params user.ResetPasswordParams) middleware.Responder {
+  fmt.Println("entered!")
+  body := params.ReqBody
+  db := config.GetDB()
+  var u models.User
+  fmt.Println(body.Username + ", " + body.Password + ", " + body.NewPassword)
```
Comment (XiangFrank) [inline tides-server/pkg/handler/user_api.go]: don't print out the username and password, if this is for test, please delete it

File: tides-server/pkg/handler/user_api.go [hunk 2]
```diff
@@ -10,7 +12,45 @@ import (
 
 	"tides-server/pkg/config"
 	"tides-server/pkg/models"
+
+	"crypto/tls"
+	gomail "gopkg.in/mail.v2"
+	"github.com/sethvargo/go-password/password"
 )
+const OFFICIAL_EMAIL = "cloudtest2021abc@gmail.com"
+const OFFICIAL_PASSWORD = "cloudtides"
```
Comment (XiangFrank) [inline tides-server/pkg/handler/user_api.go:21]: it is not recommended to hard coded any password in the code, if this email and password is very important, please remove this part and try to use environment variable.

File: tides-server/pkg/models/vapp.go [hunk 3]
```diff
@@ -36,6 +36,9 @@ type Vapp struct {
 	Status string `json:"status"`
 
 	VMs []VMachine
+
+	// vappID
+	VappID uint `gorm:"primary_key" json:"vappID,omitempty"`
```
Comment (XiangFrank) [inline tides-server/pkg/models/vapp.go:41]: Vapp already have a primarykey, why add this one.

File: tides-server/pkg/handler/user_api.go (hunk 2 above)
Comment (Kitatine) [reply tides-server/pkg/handler/user_api.go:21]: @yvvym for resolve

File: tides-server/pkg/handler/user_api.go (hunk 1 above)
Comment (Kitatine) [reply tides-server/pkg/handler/user_api.go]: @yvvym to resolve the issue

**Pull request closed without merging**